app/sftp.py

```diff
@@ -0,0 +1,39 @@
+import paramiko
+
+from config import Config
+
+
+class SftpUtility:
+
+    def __init__(self):
+        self.sftp_directory = Config.SFTP_DIRECTORY
+        self.ssh_client = paramiko.SSHClient()
+        self.ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        self.ssh_client.connect(hostname=Config.SFTP_HOST,
+                                port=int(Config.SFTP_PORT),
+                                username=Config.SFTP_USERNAME,
+                                key_filename=Config.SFTP_KEY_FILENAME,
+                                passphrase=Config.SFTP_PASSPHRASE,
+                                look_for_keys=False,
+                                timeout=120)
+
+    def __enter__(self):
+        self._sftp_client = self.ssh_client.open_sftp()
+        if not self.sftp_directory_exists:
+            self._sftp_client.mkdir(self.sftp_directory)
+            print(f'Created new directory on SFTP remote {self.sftp_directory}')
```

Comment:
Add logger at top of file and:

```suggestion
            logger.info('Created new directory on SFTP remote', sftp_directory=self.sftp_directory)
```